tests/integration/routes/profile/patch-profile-test.js

```diff
@@ -15,9 +15,9 @@ var routeOptions = {
   },
   payload: {
     data: {
-      id: 'userid123-profile',
       type: 'profile',
       attributes: {
+        id: 'userid123-profile',
```

Comment:
One of those "it made it pass the test" scenarios... I've instead fixed it properly since!